src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -1,43 +1,113 @@
+                    </a>
+                </div>
+            </div>
+            <div class="column-one-quarter">
+                <div class="data lottery-ticket">
+                    <a asp-route="@RouteNames.Dashboard_Index_Get" asp-route-statusFilter="@VacancyStatus.Submitted.GetDisplayName()" class="prototype-link">
+                        <h2 class="bold-xlarge">@Model.NoOfSubmittedVacancies</h2>
+                        <p class="bold-xsmall">@VacancyStatus.Submitted.GetDisplayName()</p>
+                    </a>
+                </div>
+            </div>
+            <div class="column-one-quarter">
+                <div class="data lottery-ticket">
+                    <a asp-route="@RouteNames.Dashboard_Index_Get" asp-route-statusFilter="@VacancyStatus.Referred.GetDisplayName()" class="prototype-link">
+                        <h2 class="bold-xlarge">@Model.NoOfEditsRequiredVacancies</h2>
+                        <p class="bold-xsmall">@VacancyStatus.Referred.GetDisplayName()</p>
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="grid-row">
+            <div class="column-one-quarter">
+                <div class="data lottery-ticket">
+                    <a asp-route="@RouteNames.Dashboard_Index_Get" asp-route-statusFilter="@VacancyStatus.Live.GetDisplayName()" class="prototype-link">
+                        <h2 class="bold-xlarge">@Model.NoOfLiveVacancies</h2>
+                        <p class="bold-xsmall">@VacancyStatus.Live.GetDisplayName()</p>
+                    </a>
+                </div>
+            </div>
+            @*<div class="column-one-quarter">
+                    <div class="data lottery-ticket">
+                        <a href="#1" class="prototype-link">
+                            <h2 class="bold-xlarge">2</h2>
+                            <p class="bold-xsmall">Closed</p>
+                        </a>
+                    </div>
+                </div>
+                <div class="column-one-quarter">
+                    <div class="data lottery-ticket">
+                        <a href="#1" class="prototype-link">
+                            <h2 class="bold-xlarge">0</h2>
+                            <p class="bold-xsmall">Archived</p>
+                        </a>
+                    </div>
+                </div>*@
+        </div>
+    </div>
     <div class="column-full">
         <table>
             <thead>
                 <tr>
                     <th class="bold">Vacancy title</th>
+                    <th class="bold">Reference number</th>
                     <th class="bold">Status</th>
                 </tr>
             </thead>
             <tbody>
-                @foreach (var vacancy in Model.Vacancies)
-                {
+@foreach (var vacancy in Model.DisplayVacancies)
+{
+    var isMissingVacancyReferenceNo = !vacancy.HasVacancyReference;
```

Comment:
I don't think this variable adds any value

Reply:
It is used to display `Not available` for any vacancy that hasn't has a ref no assigned.

Reply:
I meant I don't see much enough improvement to warrant the creation of a new variable:
`<span asp-condition="@isMissingVacancyReferenceNo">Not available</span>`
vs
`<span asp-condition="@!vacancy.HasVacancyReference">Not available</span>`

Reply:
I am not going to change this. I think it reads cleaner and this is subjective.

Reply:
👍